Directory.Solution.targets

```diff
@@ -7,7 +7,7 @@
         <Exec Command="docker run -d -p $(_RegistryPort):5000 --restart=always --name registry registry:2" />
     </Target>
 
-    <Target Name="PreloadBaseImages">
+    <Target Name="PreloadBaseImages" DependsOnTargets="StartDockerRegistry">
```

Comment:
@baronfel this seemed more straightforward to us but I'm not super tied to it if you think they'd make more sense separated.

Reply:
That's fine by me. The goal is to be able to remove this target entirely though at some point :)